src/styles/index.scss

```diff
@@ -1,3 +1,87 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: wrap;
+  flex-direction: row;
```

Comment:
According to the checklist, `flex-flow` is a shorthand property for `flex-direction` and `flex-wrap`, and using `flex-flow: row wrap;` would be more concise than declaring `flex-direction` and `flex-wrap` separately.